Dockerfile

```diff
@@ -8,18 +8,17 @@ WORKDIR /ghostfolio
 COPY ./CHANGELOG.md CHANGELOG.md
 COPY ./LICENSE LICENSE
 COPY ./package.json package.json
-COPY ./yarn.lock yarn.lock
-COPY ./.yarnrc .yarnrc
+COPY ./package-lock.json package-lock.json
 COPY ./prisma/schema.prisma prisma/schema.prisma
 
-RUN apt update && apt install -y \
-    g++ \
-    git \
-    make \
-    openssl \
-    python3 \
-    && rm -rf /var/lib/apt/lists/*
-RUN yarn install
+RUN apt-get update && apt-get install -y \
```

Comment:
Why `apt-get` instead of `apt`?

Reply:
Both of can be used but from what I have seen in popular images on Dockerhub and Docker usually prefer apt-get since it's older and stable. apt is newer and generally stable. If you want I can leave it as it is as apt install.